src/popup/router/routes/index.js

```diff
@@ -35,7 +35,7 @@ import Retip from '../pages/Retip.vue';
 import SecuritySettings from '../pages/SecuritySettings.vue';
 import Transfer from '../pages/Transfer.vue';
 import TransferSend from '../pages/TransferSend.vue';
-import TransferReceive from '../pages/TransferReceive.vue';
+import TransferReceive from '../components/Modals/TransferReceive.vue';
```

Comment:
Using a modal page as a route component will break the possibility of closing the modal. This component needs to be wrapped with another one that will manage the open state and react on `close` event when user clicks "X" or somewhere around the modal.